src/config.ts

```diff
@@ -14,7 +14,6 @@ import { TemplateOption } from './templates'
 
 const lstat = promisify(fs.lstat)
 
-type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
```

Comment:
TypeScript 3.5 already has provided `Omit` type helper.
https://devblogs.microsoft.com/typescript/announcing-typescript-3-5/#the-omit-helper-type